docs/getting-started/user-guides/paymetheus.md

```diff
@@ -17,7 +17,8 @@ You're now ready to start using Decred! Start the 'Decred' program. You will now
 ![Paymetheus connection screen](../../img/Paymetheus-dcrd-login.png)  
 
 >Two programs are actually running when you start Decred. There's 'Paymetheus' which is the graphical wallet we'll talk further about
->and 'dcrd' which is the communications program that actually talks to the Decred network. This means that you can run dcrd
+>and 'dcrd' which is the communications daemon (pronounced DAY-mon; a program that runs in the background and that does not interact
```

Comment:
That's why I suggested adding the word 'directly' since users do interact with it, just through other things.